### PR TITLE
translations: add initial portuguese translations

### PR DESCRIPTION
**Pull request by lucasmoura**
## Why is this needed?
Add portuguese translations to all the commands we support. We still need to translate many service specific messages here

## Test Steps
Guarantee that the portuguese messages are correct and properly showing up in commands

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?

 - [ ] Yes
 - [x] No

### CODE REVIEW

File: debian/po/pt_BR.po [hunk 1]
```diff
@@ -231,69 +275,82 @@ msgstr ""
...
 #: ../../uaclient/messages/__init__.py:49
 msgid "Interrupt received; exiting."
-msgstr ""
+msgstr "Sinal de interrupção recebido; saindo."
 
 #: ../../uaclient/messages/__init__.py:51
 #, python-brace-format
 msgid "Operation in progress: {lock_holder} (pid:{pid})"
-msgstr ""
+msgstr "Operação em andamento: {lock_holdedr} (pid:{pid})"
```
Comment (orndorffgrant) [inline debian/po/pt_BR.po:287]: typo
```suggestion
msgstr "Operação em andamento: {lock_holder} (pid:{pid})"
```

File: features/i18n.feature [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Feature: Pro supports multiple languages
 
+    @wip
```
Comment (orndorffgrant) [inline features/i18n.feature:3]:
```suggestion
```

**Review by orndorffgrant — COMMENTED** [review]
This is amazing :rocket:

**Comment by renanrodrigo** [comment]
@lucasmoura I have improved it, fixing typos and some punctiation, and some concordancy or whatever.
There is still things to define and improve but I'd say this is sane now.
Please take a look on the commit, and if you agree, well
:shipit:

File: debian/po/pt_BR.po [hunk 3]
```diff
@@ -856,7 +856,7 @@ msgid ""
 "Service is still active. See `pro status`"
 msgstr ""
 "Falha ao desabilitar '{service}' conforme esperado após atualização do seu "
-"contratoServiço ainda está ativo. Execute `pro status` para confirmar"
+"contrato. Os serviço ainda está ativo. Execute `pro status` para confirmar"
```
Comment (lucasmoura) [inline debian/po/pt_BR.po:859]: s/Os/O/

Comment (renanrodrigo) [reply debian/po/pt_BR.po]: fixed

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**